source/imaer-gml/src/test/java/nl/overheid/aerius/gml/base/geo/Geometry2GMLTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright the State of the Netherlands
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+package nl.overheid.aerius.gml.base.geo;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import net.opengis.gml.v_3_2_1.DirectPositionType;
+import net.opengis.gml.v_3_2_1.PointType;
+
+import nl.overheid.aerius.gml.base.GMLSchema;
+import nl.overheid.aerius.shared.domain.v2.geojson.Point;
+import nl.overheid.aerius.shared.exception.AeriusException;
+import nl.overheid.aerius.shared.geo.EPSG;
+
+class Geometry2GMLTest {
+  private final Geometry2GML geo2gml = new Geometry2GML(GMLSchema.getSRSName(EPSG.RDNEW.getSrid()));
+
+  private GmlPoint createMockPoint() {
+    return new GmlPoint() {
+      private PointType gmlPoint = null;
+
+      @Override
+      public PointType getGmlPoint() {
+        return gmlPoint;
+      }
+
+      @Override
+      public void setGmlPoint(final PointType gmlPoint) {
+        this.gmlPoint = gmlPoint;
+      }
+    };
+  }
+
+  static Stream<Arguments> pointProvider() {
+    return Stream.of(
+        Arguments.of("Rounding up", new Point(5.00015, -5.00015), "5.0", "-5.0"),
+        Arguments.of("Rounding down", new Point(5.00014, -5.00014), "5.0", "-5.0"),
```

Comment:
ehhhh what is rounding up/rounding down in this case? Shouldn't it be `5.0015` and `5.0014` if you want to test that?

And if not, might be nice to add a case that does something like `1.23456789` getting rounded to `1.235`.

Reply:
```
        Arguments.of("Rounding up positive", new Point(1.0001, -0.9999), "1.0", "-1.0"),
        Arguments.of("Rounding down negative", new Point(-0.9999, 1.0001), "-1.0", "1.0"),
        Arguments.of("Edge case rounding up", new Point(0.9995, -0.9995), "1.0", "-0.999"),
        Arguments.of("Edge case rounding down", new Point(0.9994, -0.9994), "0.999", "-0.999"),
```

I've added them like this (hopefully also very readable)